client/map_verifier.go

```diff
@@ -49,7 +51,17 @@ func NewMapVerifierFromTree(config *trillian.Tree) (*MapVerifier, error) {
 		return nil, fmt.Errorf("Failed parsing Map public key: %v", err)
 	}
 
-	return &MapVerifier{Hasher: mapHasher, PubKey: mapPubKey}, nil
+	// Map Sig Hash
```

Comment:
(as log)

Reply:
Removed comments so as not to be redundant.